docs/unity_specific.md

```diff
@@ -39,7 +39,9 @@ Similarly we can feed the `IntPtr` received from `LoadLibrary(string)` into `Fre
 Hence allowing the sdk to modify its configurations.
 
 
-### Reference to other documents on how and why we need a GFX plugin on windows
+### Why do we need a GFX plugin on Windows
+
+GFX plugin is required so that the EOS Overlay can render properly on Windows
```

Comment:
* On line 14, it should read "has to be compatible with" instead of "has to compat with"
* Line 26 "rerun the plugin in without" should be "return the plugin without"
* The sentence on line 44 needs a period. 

As a general note, we may not need to talk about how we LoadLibrary / FreeLibrary.